src/Traits.h

```diff
@@ -1007,16 +1007,45 @@ struct Traits<std::chrono::system_clock::duration> : public Impl::CommonDuration
 template <typename ClockT, typename DurationT>
 struct Traits<std::chrono::time_point<ClockT, DurationT>> : public TraitsImpl<std::chrono::time_point<ClockT, DurationT>> {
     static std::string ToString(std::chrono::time_point<ClockT, DurationT> const &tp) {
-        std::ostringstream                  out;
-
-        date::operator <<(out, tp);
-        out.flush();
-
-        return out.str();
+        return date::format("%FT%TZ", date::floor<std::chrono::seconds>(tp));
     }
 
     static std::chrono::time_point<ClockT, DurationT> FromString(std::string const &value) {
-        std::ignore = value; throw std::logic_error("Not Implemented Yet");
+        std::string dash = "";

```

Comment:
I don't believe that these values will be necessary once you use std::ostringstream.

Reply:
Now that I read more of the code, I see that you are using these values to populate the date_template. Update these names to indicate that this is how they will be used "dash_template_component", "colon_template_component", etc.